### ISSUE TITLE
Sonic Rivals 2 too bright on some machines

### ISSUE THREAD

**Issue by hrydgard** [hrydgard/ppsspp#18015]
Seems https://github.com/hrydgard/ppsspp/issues/6117 has returned in some form, not sure what's going on.

Reported in that issue, and on Discord by Feilakas.

**Comment by Duglor** [comment]
Is this fixed as of the 1.16 release?

**Comment by hrydgard** [comment]
I don't think so, but also I've been completely unable to reproduce this error on any hardware I have. It just works for me.

**Comment by hrydgard** [comment]
May be related to #17680, i'll take a look.

**Comment by hrydgard** [comment]
Haven't gotten any more reports of this, I'll assume it's fixed. The game appears to work.
